test/deprecated/atomic/return-by-ref.compopts

```diff
@@ -0,0 +1,18 @@
+-stestnum=1 -sretType='atomic int' # return-by-ref-warning.good
+-stestnum=2 -sretType='atomic int' # return-by-ref-warning.good
+-stestnum=3 -sretType='atomic int'
+-stestnum=1 -sretType='[0..2] atomic int' # return-by-ref-warning.good
+-stestnum=2 -sretType='[0..2] atomic int' # return-by-ref-warning.good
+# -stestnum=3 -sretType='[0..2] atomic int'
```

Comment:
Since this and line 9 are commented out (and look to be handled by return-by-ref-array.compopts), can they be deleted?